index.html

```diff
@@ -17,8 +17,8 @@
 			<header id="header">
 				<div class="inner">
 					<div class="content">
-						<h1>Alpha</h1>
-						<h2>A gallery of unique photos.</h2>
+						<h1>Welcome my friends!</h1>
```

Comment:
Hi Dom, please remove this change, because index.html should be a page that describes our entire gallery. (not just a single person) Other changes in index.html is unecessary, we only need to add your image in this page. :) thanks